modules/api/src/test/java/org/apache/ignite/lang/IgniteCheckedExceptionTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.lang;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.ignite.lang.ErrorGroups.Common;
+import org.junit.jupiter.api.Test;
+
+/** For {@link IgniteCheckedException} testing.  */
+public class IgniteCheckedExceptionTest {
+    @Test
+    void testInternalErr() {
```

Comment:
Please do not use abbreviations in method names here and below.

Reply:
I don't think the comment is correct in this case since the error code already contains an abbreviation, in any case I rescheduled the test.

Reply:
> since the error code already contains an abbreviation

There is quite a small difference between method names and public static fields. For instance, Apache Ignite 2 states that `Type and method names are usually not abbreviated (except for the well-accepted abbreviations such as EOF, Impl, Fifo, etc.).` and I still think that this rule is a good one :)

Anyway, thank you for the change.